tests/test_2fa.py

```diff
@@ -1,16 +1,29 @@
+import random
 import secrets
+import string
+from typing import TypeAlias
 
 import pyotp
+import pytest
 from auth_helper import register_user_2fa
 from flask.testing import FlaskClient
 
+LoginData: TypeAlias = dict[str, str]
+
+@pytest.fixture()
+def nonce(len: int = 7) -> str:
```

Comment:
Let's bump up the nonce length just to make sure there's no duplicates. I'd recommend 16.

Reply:
addressed by [894a61d](https://github.com/scidsg/hushline/pull/460/commits/894a61d0a16545e7ce55fbe6dd980f6e1aa45903)